server/ladder_service.py

```diff
@@ -161,7 +162,13 @@ def _cancel_existing_searches(self, initiator: Player) -> List[Search]:
         # FIXME: Database filenames contain the maps/ prefix and .zip suffix.
         # Really in the future, just send a better description
         self._logger.debug("Starting ladder game: %s", game)
-        await host.lobby_connection.launch_game(game, is_host=True, use_map=mapname)
+        options = GameLaunchOptions(
+            mapname=mapname,
+            expected_players=2
+        )
```

Comment:
The _unused_ options "team", "faction", and "map_position" are now no longer set,
before they were coming from `game.get_player_options`.
Since they are unused I assume that is no problem, I just wanted to make sure this is as intended.

If they are ever used, is the correct place to include them here?
If _not_, then I would move them back into the lobbyconnection.
I guess @norraxx knows what he would want, since he is working on tmm?